src/marqo/s2_inference/processing/image.py

```diff
@@ -211,9 +211,11 @@ def _load_and_cache_model(self):
 
             self.model, self.preprocess = func(self.model_name, self.device)
 
-            available_models[model_cache_key] = self.model, self.preprocess
+            available_models[model_cache_key] = {"model": (self.model, self.preprocess),
+                                                 "time_stamp" : datetime.datetime.now()}
```

Comment:
Is this better as `most_recently_used_time`? `time_stamp` doesn't explain exactly what this is for